content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -318,22 +295,19 @@ The following networking functionality is not supported on Windows nodes
   * ICMP packets directed to pass through a remote network (e.g. pod to external internet communication via ping) cannot be transposed and thus will not be routed back to their source
   * Since TCP/UDP packets can still be transposed, one can substitute `ping <destination>` with `curl <destination>` to be able to debug connectivity to the outside world.
 
-These features were added in Kubernetes v1.15:
+#### CNI plugins
 
-* `kubectl port-forward`
-
-##### CNI Plugins
-
-* Windows reference network plugins win-bridge and win-overlay do not currently implement [CNI spec](https://github.com/containernetworking/cni/blob/master/SPEC.md) v0.4.0 due to missing "CHECK" implementation.
-* The Flannel VXLAN CNI has the following limitations on Windows:
+* Windows reference network plugins win-bridge and win-overlay do not implement [CNI spec](https://github.com/containernetworking/cni/blob/master/SPEC.md) v0.4.0, due to a missing "CHECK" implementation.
+* The Flannel VXLAN CNI plugin has the following limitations on Windows:
 
 1. Node-pod connectivity isn't possible by design. It's only possible for local pods with Flannel v0.12.0 (or higher).
-2. We are restricted to using VNI 4096 and UDP port 4789. The VNI limitation is being worked on and will be overcome in a future release (open-source flannel changes). See the official [Flannel VXLAN](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#vxlan) backend docs for more details on these parameters.
+2. Flannel is restricted to using VNI 4096 and UDP port 4789. See the official [Flannel VXLAN](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#vxlan) backend docs for more details on these parameters.
```

Comment:
Might be useful to call out https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere in the context of port 4789.

Reply:
That might be a separate update - this one's already large.